packages/styles/scss/components/quote/_quote.scss

```diff
@@ -78,6 +78,10 @@
     @include carbon--breakpoint('lg') {
       left: -1rem;
     }
+
+    @include carbon--breakpoint('max') {
+      left: -2rem;
```

Comment:
@emyarod can we get spacing tokens in this file (wherever possible, I see a few fractions which I guess can be left that way for now).